song_translator/track/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+
+class Singer(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+class Track(models.Model):
+    name = models.CharField(max_length=64)
+    text = models.TextField()
+    translate_text = models.TextField(blank=True)
+    original_language = models.CharField(max_length=2)
```

Comment:
let's try restricting languages we support via enum https://www.postgresql.org/docs/9.1/datatype-enum.html
maybe also on serializer level